cloud/filestore/libs/vfs_fuse/fs_impl_data.cpp

```diff
@@ -367,6 +414,18 @@ void TFileSystem::Write(
     request->SetBufferOffset(alignedBuffer.AlignedDataOffset());
     request->SetBuffer(alignedBuffer.TakeBuffer());
 
+    if (Config->GetServerWriteBackCacheEnabled()) {
+        if (!WriteBackCache->AddWriteDataRequest(std::move(request))) {
+            STORAGE_DEBUG(
+                "WriteBackCache overflow, can't add WriteDataRequest for "
+                << "#" << ino << " @" << fi->fh);
+            return;
+        }
+
+        ReplyWrite(*callContext, MakeError(S_OK), req, buffer.size());
```

Comment:
what if the write actually need to fail ? For example there is write to read only file or the file offset is invalid or other corner cases that we handle inside the tablet

This potentially can break many applications